src/features/getData.js

```diff
@@ -11,6 +11,9 @@ export const getMovieDetails = async ({ movieId }) => {
   return await response.data;
 };
 
+const movie = await getMovieDetails({ movieId: 926393 });
+console.log(movie.title);
+
```

Comment:
I think you forgot to delete that after testing.

Reply:
Yes, thank you ^^